engine/storage/snapshot/src/main/java/org/apache/cloudstack/storage/vmsnapshot/StorageVMSnapshotStrategy.java

```diff
@@ -449,14 +454,14 @@ protected SnapshotInfo createDiskSnapshot(VMSnapshot vmSnapshot, List<SnapshotIn
         return snapshotInfo;
     }
 
-    protected CreateSnapshotPayload setPayload(VolumeInfo vol, SnapshotVO snapshotCreate) {
+    protected CreateSnapshotPayload setPayload(VolumeInfo vol, SnapshotVO snapshotCreate, boolean quiescevm) {
         CreateSnapshotPayload payload = new CreateSnapshotPayload();
         payload.setSnapshotId(snapshotCreate.getId());
         payload.setSnapshotPolicyId(SnapshotVO.MANUAL_POLICY_ID);
         payload.setLocationType(snapshotCreate.getLocationType());
         payload.setAccount(accountService.getAccount(vol.getAccountId()));
         payload.setAsyncBackup(false);
-        payload.setQuiescevm(false);
+        payload.setQuiescevm(quiescevm);
```

Comment:
@sureshanaparti this payload is being used to set in SnapshotObjecTO, but it is not used anywhere. or am I missing anything ?

Reply:
It is passed with the payload in the snapshotInfo, the underlying storage (if supports) would make use of it.